scripts/dev_setup.sh

```diff
@@ -628,7 +628,7 @@ function install_nodejs {
 }
 
 function install_pnpm {
-    curl -fsSL https://get.pnpm.io/install.sh | "${PRE_COMMAND[@]}" env PNPM_VERSION=7.14.2 bash -
+    curl -fsSL https://get.pnpm.io/install.sh | "${PRE_COMMAND[@]}" env PNPM_VERSION=7.14.2 SHELL=$(which "$SHELL") "$SHELL" -
```

Comment:
This is bash specific syntax, `curl -fsSL https://get.pnpm.io/install.sh | "${PRE_COMMAND[@]}" env PNPM_VERSION=7.14.2 "$SHELL"` should cover all shells unless your shell is not on your path (which probably would make things very confusing)

Reply:
Thx! Let me look into it.

Reply:
`curl -fsSL https://get.pnpm.io/install.sh | "${PRE_COMMAND[@]}" env PNPM_VERSION=7.14.2 "$SHELL"`, this works well. I'll update the code. Ty!

Reply:
It didn't seem to work for me :(